config/jail.conf

```diff
@@ -47,7 +47,7 @@ before = paths-debian.conf
 # "ignoreip" can be an IP address, a CIDR mask or a DNS host. Fail2ban will not
 # ban a host which matches an address in this list. Several addresses can be
 # defined using space (and/or comma) separator.
-ignoreip = 127.0.0.1/8
+ignoreip = 127.0.0.1/8 ::1
```

Comment:
To add *::1* reflects just the default value for IPv4 and gives a hint, that also IPv6 addresses or networks can be added to this configuration variable.
I don't know why *fail2ban*'s IPv6 support should be disabled. If it runs on a host with IPv4 connectivity only, then only the IPv4 specific parts of the code are being executed. The presence of the only depended functions **inet_pton()** and **inet_ntop()** are bound to the *python* version. This could be the only decision point IMHO.
Generally my personal opinion on this is, that the time is over to consider IPv6 being optional in network related software.